index.js

```diff
@@ -88,17 +76,21 @@
 //. var Identity = require('my-package').Identity;
 //. var type = require('sanctuary-type-identifiers');
 //.
-//. type(null);         // => 'Null'
-//. type(true);         // => 'Boolean'
-//. type([1, 2, 3]);    // => 'Array'
-//. type(Identity);     // => 'Function'
-//. type(Identity(0));  // => 'my-package/Identity'
+//. type(null);        // => {namespace: null, name: 'Null', version: null}
+//. type(true);        // => {namespace: null, name: 'Boolean', version: null}
+//. type([1, 2, 3]);   // => {namespace: null, name: 'Array', version: null}
+//. type(Identity);    // => {namespace: null, name: 'Function', version: null}
+//. type(Identity(0)); // => {
+//.                    //      namespace: 'my-package',
+//.                    //      name: 'Identity',
+//.                    //      version: null
+//.                    //    }
```

Comment:
I think these would read better as doctests. Could you add [doctest][1] as a dependency and invoke it from the `test` target in the makefile?


[1]: https://github.com/davidchambers/doctest